src/leap/mail/generator.py

```diff
@@ -0,0 +1,20 @@
+from email.generator import Generator as EmailGenerator
+
+class Generator(EmailGenerator):
+    """
+    Generates output from a Message object tree, keeping signatures.
+    
+    This code was extracted from Mailman.Generator.Generator, version 2.1.4:
+    
+    Most other Generator will be created not setting the foldheader flag,
+    as we do not overwrite clone(). The original clone() does not
+    set foldheaders.
+
+    So you need to set foldheaders if you want the toplevel to fold headers
+    This class was extracted from
+    """
```

Comment:
Can you add a XXX or TODO comment mentioning the version of python that solves this issue.